libs/dh/shared/feature-authorization/src/lib/dh-selected-actor.component.ts

```diff
@@ -35,25 +35,29 @@ export type SelectionMarketParticipant = ResultOf<
   typeof GetSelectionMarketParticipantsDocument
 >['selectionMarketParticipants'][0];
 
+const meteringPointSubPathRegex = /^\/metering-point\/\d+\//;
+
 @Component({
   selector: 'dh-selected-actor',
   styleUrls: ['./dh-selected-actor.component.scss'],
   templateUrl: './dh-selected-actor.component.html',
   imports: [WattIconComponent, WattSpinnerComponent, OverlayModule, TranslocoPipe],
 })
 export class DhSelectedActorComponent {
-  private location = inject(windowLocationToken);
-  private actorStorage = inject(DhActorStorage);
+  private readonly router = inject(Router);
+  private readonly location = inject(windowLocationToken);
+  private readonly actorStorage = inject(DhActorStorage);
 
-  selectedMarketParticipants = query(GetSelectionMarketParticipantsDocument);
+  private query = query(GetSelectionMarketParticipantsDocument);
+  private invitedToMarketParticipants = computed(
```

Comment:
Suggestion: memberOfMarketParticpants might be a better name?

Reply:
Good idea. Fixed :)